app/config/config_test.yml

```diff
@@ -15,7 +15,6 @@ framework:
 doctrine:
     dbal:
         dbname: "%database_name%_test"
-        driver_class: Sylius\Bundle\CoreBundle\Test\MySqlDriver
```

Comment:
seems this is a different class than the one I deleted here: https://github.com/Sylius/Sylius/pull/7497/files#diff-89e907faa9c86340f8a0ae7df6ecdfaaL12